core/lib/config/src/configs/api.rs

```diff
@@ -46,6 +46,107 @@ impl ApiConfig {
     }
 }
 
+/// Port binding specification.
+///
+/// Supports any of 3 formats:
+///
+/// - Just a `u16` port. This will bind a server to all IPv4 interfaces (i.e., `0.0.0.0`) for backward compatibility.
+/// - Full socket address (e.g., `127.0.0.1:8080`).
+/// - (For Unix systems) Path to a Unix domain socket (UDS) prefixed by `ipc:` (e.g., `ipc:chains/era/health.sock` or `ipc:/var/zksync.sock`).
```

Comment:
Bikeshedding on syntax appreciated. AFAIU, there's no standardized spec, but some other apps use `ipc://` prefix (i.e., `ipc://./chains/era/health.sock` or `ipc:///var/zksync.sock`).

Reply:
`reth` and `anvil` use `/tmp/{reth,anvil}.ipc` by default. That being said they provide full JSON RPC API through that socket. Tangential to this PR but maybe we should do the same for the sake of minimal latency in integration tests? `reth` has a `jsonrpsee` middleware that we can likely adapt for our needs.

As for the name I like the `ipc://` prefix (btw the rustdoc uses `ipc:` instead, not sure if its intentional). I think its reasonable to assume that there is one server per machine just like we assume default port without taking chain into account. So I would use one of these: `ipc:///run/zksync.sock`, `ipc:///var/run/zksync.sock`, `ipc:///tmp/zksync.sock`. Not sure if there is a difference between 1 and 2 in modern Linux, I have seen both conventions being used.